packages/locale/lang/uz-uz.ts

```diff
@@ -120,8 +120,8 @@ export default {
       title: 'Orqaga',
     },
     popconfirm: {
-      confirmButtonText: 'Yes', // to be translated
-      cancelButtonText: 'No', // to be translated
+      confirmButtonText: 'Ha',
+      cancelButtonText: 'Yo`q',
```

Comment:
refer to https://github.com/ElemeFE/element/blob/290e68ea6aa6c56b7d83182b650e3be4f77ff1b0/src/locale/lang/uz-UZ.js#L116C27-L116C37